packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -490,6 +501,11 @@ class WebViewController {
     return _channel.invokeMethod("reload");
   }
 
+  /// Accessor the UserAgent.
```

Comment:
Please follow the [documentation section in the Flutter style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc), I'd suggest:

```
/// Returns the User-Agent value that will be used for subsequent HTTP requests.
```